acquamanCommon.pri

```diff
@@ -3,7 +3,7 @@
 # ####################################################################
 
 # Video Support: Remove this line if you do not have the multimedia module from QtMobility
-CONFIG *= mobility
+#CONFIG *= mobility
```

Comment:
Including mobility seemed to be causing me compile problems. Not sure if it was because of SL7. I initially tried to remove it from the .pro file of the driver server (to keep the change as limited as possible), but that didn't seem to solve the issue. It was only when I put it here that it finally compiled. I have a checklist of things that I've done to get things working for now, but that will need to be resolved before this is merged into master.